packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -201,7 +215,11 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
                 quoteTimestamp: quoteTimestamp
             });
         bytes32 depositHash = _getDepositHash(depositData);
-        require(relays[depositHash].relayState == RelayState.Uninitialized, "Pending relay for deposit exists");
+        require(
```

Comment:
This change explained [above](https://github.com/UMAprotocol/protocol/pull/3309/files#r689692690)